pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
     "scipy >= 1.4.0",
     "tqdm >= 4.32.0",
     "woodwork[dask] >= 0.18.0",
+    "typing_extensions == 4.5.0",
```

Comment:
Not a fan of adding dependencies to Featuretools, unless it's absolutely necessary.